data/books/ro.json

```diff
@@ -0,0 +1,265 @@
+{
+    "GEN": [
+        "geneza",
+        "gen"
+    ],
+    "EXO": [
+        "exodul",
+        "exod"
+    ],
+    "LEV": [
+        "levitic",
+        "lev"
+    ],
+    "NUM": [
+        "numeri",
+        "num"
+    ],
+    "DEU": [
+        "deuteronom",
+        "deu"
+    ],
+    "IOS": [
+        "iosua",
+        "ios"
+    ],
+    "JUD": [
+        "judecatori",
+        "jud"
+    ],
+    "RUT": [
+        "ruth",
+        "rut"
+    ],
+    "1SA": [
+        "1samuel",
+        "1sa"
+    ],
+    "2SA": [
+        "2samuel",
+        "2sa"
+    ],
+    "1IMP": [
+        "1împărați",
+        "1imparati",
+        "1imp",
+        "1împ"
+    ],
+    "2IMP": [
+        "2împărați",
+        "2imparati",
+        "2imp",
+        "2împ"
+    ],
+    "1CRON": [
```

Comment:
Example of invalid key

Reply:
indeed, I went out of my way yesterday to change the keys also to romanian, I now see they should be left in english